src/terminal/terminal/terminal.h

```diff
@@ -591,15 +172,6 @@ typedef struct guac_terminal_options {
      */
     guac_client* client;
```

Comment:
I didn't notice this in the previous iteration, but I don't think this should be an "option" either. It's the sort of thing that would be provided at terminal creation and then never change.